examples/todo/.eslintignore

```diff
@@ -0,0 +1,5 @@
+node_modules
```

Comment:
could maybe use the same one as in base. or even just have the base package handle linting here